src/Esfa.Vacancy.Register.Api/Controllers/GetApprenticeshipVacancyController.cs

```diff
@@ -46,7 +46,7 @@ public GetApprenticeshipVacancyController(GetApprenticeshipVacancyOrchestrator a
         /// </summary>
         [HttpGet]
         [AllowAnonymous]
-        [Route("{vacancyReference}", Name = RouteName.GetApprenticeshipVacancyByReference)]
+        [Route("{vacancyReference:regex(^(?i)(?!search).*$)}", Name = RouteName.GetApprenticeshipVacancyByReference)]
```

Comment:
Just a suggestion, we could change the route prefix for search, instead of /apprenticeships/search we could use /apprenticeshipssearch

Reply:
Noted. Let's discuss as we will have issues like this in the future as well. This was just to fix what is broken at the moment.